src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,20 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => {
+  const { completed, title, user } = todo;
+
+  return (
+    <article
+      className={`TodoInfo ${completed
```

Comment:
The ternary operation should be simplified for readability. Consider creating a helper function or using classnames library.